app/assets/stylesheets/_govuk-elements.scss

```diff
@@ -17,3 +12,7 @@ $path: false; // `elements/helpers` sets this to `public/images` which breaks
 //@import "elements/details"; - not yet needed
 //@import "elements/panels";  - not yet needed
 //@import "elements/icons";   - not yet needed
+
+$path: false; // `elements/helpers` sets this to `public/images` which breaks
```

Comment:
Moving this down will break the button assets - I've spun up a review app from this branch to show this - https://warm-journey-3915.herokuapp.com/appointments